### PR TITLE
Enhancement/skale 2364 merge monitors

### PR DESCRIPTION
**Pull request by yavrsky**
Merged MonitorsData with MonitorsFunctionality, GroupsData with GroupsFunctionalty

### CODE REVIEW

File: test/MonitorsData.ts [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
-import { ContractManagerInstance, MonitorsDataInstance } from "../types/truffle-contracts";
+import { ContractManagerInstance, MonitorsInstance } from "../types/truffle-contracts";
 import { deployContractManager } from "./tools/deploy/contractManager";
-import { deployMonitorsData } from "./tools/deploy/monitorsData";
+import { deployMonitors } from "./tools/deploy/monitors";
 
 chai.should();
 chai.use(chaiAsPromised);
 
 contract("MonitorsData", ([owner, user]) => {
```
Comment (DimaStebaev) [inline test/MonitorsData.ts]: Remove this file.

**Review by payvint — DISMISSED** [review]
Wow, finally - very necessary

**Review by payvint — APPROVED** [review]
Good!

**Comment by DimaStebaev** [comment]
Close this pull request because all changes are part of https://github.com/skalenetwork/skale-manager/pull/245

**Pull request closed without merging**